index.html

```diff
@@ -58,7 +60,8 @@ <h1>Ricardo Egas</h1>
              
             </section>
 
- <div id="allProjects"></div>
+ <div id="allProjects">
+ </div>
          
     <h2 class="Project-Mobile">Projects</h2>
          
```

Comment:
- [ ] Good job creating a pop-up for each of  your project 👍 , everything is great , however I noticed that when the pop-up is opened the user is not able to all the page because the pop-up overflows 😄 , Kindly make sure that in this case the user can scroll to see all the pop-up window 👍 

![Uploading Capture1.PNG…]()